omegaconf/nodes.py

```diff
@@ -87,11 +82,9 @@ def _deepcopy_impl(self, res: Any, memo: Dict[int, Any]) -> None:
 
     def _is_none(self) -> bool:
         if self._is_interpolation():
-            node = self._dereference_node(
-                throw_on_resolution_failure=False, throw_on_missing=False
-            )
+            node = self._dereference_node(throw_on_resolution_failure=False)
             if node is None:
-                # missing or resolution failure
+                # resolution failure
                 return False
         else:
             node = self
```

Comment:
As I said, we should probably simplify the behavior of is_none as well.

Reply:
Just linking to the discussion for easy reference: https://github.com/omry/omegaconf/pull/545#discussion_r577180921